packages/mc-scripts/config/vendors-to-transpile.js

```diff
@@ -0,0 +1,6 @@
+// https://github.com/formatjs/react-intl/blob/master/docs/Upgrade-Guide.md#esm-build
+module.exports = [
+  require.resolve('react-intl'),
+  require.resolve('intl-messageformat'),
+  require.resolve('intl-messageformat-parser'),
+];
```

Comment:
I assume this won't change, right?

Reply:
It shouldn't change no. According to this comment, they plan to ship them as ESM

https://github.com/formatjs/react-intl/issues/1371#issuecomment-514255342